src/main.cpp

```diff
@@ -142,9 +157,10 @@ void usercontrol( void ) {
         // Intake Control
         if(Controller1.ButtonL1.pressing()) { //If the upper left trigger is pressed...
             //...Spin the claw motor forward.
-            IntakeMotor.spin(vex::directionType::fwd, intakeSpeedPCT, vex::velocityUnits::pct);
+            intake(intakeSpeedPCT);
         } else if(Controller1.ButtonL2.pressing()) {
-            IntakeMotor.spin(vex::directionType::fwd, -intakeSpeedPCT, vex::velocityUnits::pct);
+            //...Spin the claw motor backward.
+            intake(-intakeSpeedPCT);
         } else {
             //...Stop the arm motor.
             IntakeMotor.spin(vex::directionType::fwd, 0, vex::velocityUnits::pct);
```

Comment:
we should also use the intake method here, just with intake(0). We should also do that below with the shoot method.